config/manager/manager.yaml

```diff
@@ -39,3 +39,5 @@ spec:
       tolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
```

Comment:
There are a few more places where `node-role.kubernetes.io/master` is used. Sometimes as nodeSelector sometimes as taint.

I think they should be adjusted as well, but I'm not sure

Reply:
There is no node selector logic that CAPV uses to run the manager.

Reply:
I just found code when grepping for it, e.g.:
* cloud-controller-manager.go
* csi.go
* calico.yaml

I can't tell if those places are relevant or not. Feel free to just resolve if that finding doesn't make sense :)

Reply:
Other than the calico.yaml, that was long replaced by applying CPI/CSI using CRS. The functions should probably be marked as deprecated.